doc-content/enterprise-only/cases/case-management-defining-a-stage-proc.adoc

```diff
@@ -18,5 +18,5 @@ image::cases/itorders-stages.png[IT_Orders - stages]
 . Add the relevant tasks to the stage and configure them as required.
 . (Optional) Configure a completion condition for the stage. As an ad hoc subprocess, stages are configured as `autocomplete` by default, which means that the stage will automatically complete and trigger the next activity in the case definition once all instances in the stage are no longer active.
 +
-To change the completion condition, select the stage node and open the *Properties* panel on the right. Modify the `Completion Condition` property field in the with a free-form Drools expression for the completion condition you require. For more information about stage completion conditions, see <<case-management-stage-activation-and-completion-conditions-proc>>.
+To change the completion condition, select the stage node and open the *Properties* panel on the right, expand *Implementation/Execution*, and modify the `AdHocCompletionCondition` property field in the with a free-form Drools expression for the completion condition you require. For more information about stage completion conditions, see <<case-management-stage-activation-and-completion-conditions-proc>>.
```

Comment:
This doesn't sound right to me:

modify the AdHocCompletionCondition property field **in the** with a free-form Drools expression for the completion

Probably `in the` should be omitted?

Reply:
Fixed.